tools/SourceKit/tools/sourcekitd/include/sourcekitd/Internal.h

```diff
@@ -177,6 +179,83 @@ static inline sourcekitd_variant_t makeUIDVariant(sourcekitd_uid_t value) {
   return {{ 0, (uintptr_t)value, SOURCEKITD_VARIANT_TYPE_UID }};
 }
 
+template <typename VisitorImplClass, typename VisitedType>
+class RequestResponsePrinterBase {
```

Comment:
I recommend to split `RequestResponsePrinterBase` to its own header. `Internal.h` is getting large.